manifest.json

```diff
@@ -7,8 +7,14 @@
   "content_scripts": [
     {
       "matches": ["<all_urls>"],
-      "run_at": "document_start",
-      "js": ["content/content.js"]
+      "js": ["content/content.js"],
```

Comment:
Object key lookup shouldn't be ordered dependent, so I wonder if this actually changes anything at all? I think when I originally experimented with this I was just a victim of the data not loading, but I'm not sure.

Reply:
This shouldn't change anything for the reasons you mention.

Reply:
Yeah this change is unimportant, I can take it out, I just did this when I was trying things.